tests/conftest.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2022 Pulser Development Team
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+import pytest
+
+from pulser.channels import Raman, Rydberg
+from pulser.devices import Device
+
+
+@pytest.fixture
+def mod_device() -> Device:
```

Comment:
What about having a public ModMockDevice, or add modulation to the MockDevice?

Reply:
`MockDevice` is supposed to be irrealistically good, so I don't think it would make sense to add a HW constraint to it. A `ModMockDevice` could make sense, but then what values of bandwith would we give it?

I think the direction we are going towards is rather to give more freedom for advanced users to define their one custom devices, so they are not restrict to the public ones.